node/silkworm/downloader/internals/header_persistence.cpp

```diff
@@ -23,12 +23,12 @@
 
 namespace silkworm {
 
-PersistedChain::PersistedChain(Db::ReadWriteAccess::Tx& tx) : tx_(tx), canonical_cache_(1000) {
+HeaderPersistence::HeaderPersistence(Db::ReadWriteAccess::Tx& tx) : tx_(tx), canonical_cache_(1000) {
```

Comment:
I suggest using a constant instead of a magic number for the cache size.

Reply:
done